src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -808,6 +818,12 @@ private static PlatformAttributes CopyAllAttributes(PlatformAttributes copyTo, P
 
                 AddPlatformAttributes(symbol.GetAttributes(), ref attributes);
 
+                if (symbol is IMethodSymbol method && (method.MethodKind == MethodKind.PropertyGet || method.MethodKind == MethodKind.PropertySet))
```

Comment:
```suggestion
                if (symbol.IsAccessorMethod())
```

Reply:
This should also handle the event case...

Reply:
> This should also handle the event case...

Yeah i am just started looking that bug, looks very similar

Reply:
>  if (symbol is IMethodSymbol method && (method.MethodKind == MethodKind.PropertyGet || method.MethodKind == MethodKind.PropertySet)) => if (symbol.IsAccessorMethod())

`symbol.IsAccessorMethod()` is neat, but the result of `is` expression `method` is used within if block, so am gonna keep this as is

Reply:
You can add an extension method even on IMethodSymbol that gets called from ISymbolExtensions. It will also handle events for you.